pkgs/development/libraries/libopendmarc/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, libmilter }:
+
+stdenv.mkDerivation rec {
+  pname = "libopendmarc";
+  version = "1.3.3";
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ libmilter ];
+
+  postPatch = ''
+    substituteInPlace configure.ac --replace '	docs/Makefile' ""
+  '';
+
+  configureFlags = [
+    "--with-milter=${libmilter.out}"
```

`libmilter` only has one output, so you can just use

```suggestion
    "--with-milter=${libmilter}"
```